README.md

```diff
@@ -126,6 +126,27 @@ npm run docker:run
 # Hooray! Leon is running
 ```
 
+### Docker guide for development
+```sh
+# Clean if old node_modules and image exist
+rm -rf node_modules
+docker system prune
+docker rmi leonai/leon
```

Comment:
You don't need these commands because you don't need to remove Docker cache or older image of Leon.

Concerning `node_modules`, you don't need to remove it, if you don't do `npm  install` in the first place.